### PR TITLE
Update to readme.md

### PR DESCRIPTION
**Pull request by mackerac**
## Type of Change

- [x] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [x] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [x] I've updated the code style using `make codestyle`.
- [x] I've written tests for all new methods and classes that I created.
- [x] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
 <p align="center">
-  <img width="300" alt="giskardlogo" src="giskard-frontend/src/assets/logo_full.svg">
+  <img width="500" alt="giskardlogo" src="readme/Logo_full_darkgreen.png">
 </p>
-<h3 align="center">Deliver ML products, better & faster</h3>
+<h1 align="center" weight='300' >Open-Source CI/CD platform for ML teams. Deliver ML products, better & faster. </h1>
```
Comment (alexcombessie) [inline README.md:4]: I would keep the title just "Open-Source CI/CD platform for ML teams" and then have "Deliver ML products, better & faster" below the title, in bold.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 <br />
 
-#### Giskard is an Open-Source CI/CD platform for ML teams.
+**Collaborate faster** with **feedback** from business stakeholders.<br/>
```
Comment (alexcombessie) [inline README.md:7]: Can you add an emoji at the beginning like ⚡ ?

File: README.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+**Deploy automated tests** to eliminate regressions, errors & biases.
```
Comment (alexcombessie) [inline README.md:8]: Can you add an emoji at the beginning like 🧑‍🔧 ?

File: README.md [hunk 4]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
 
 > login / password: **admin** / **admin**
 
-<h3 align="center">Requirements</h3>
+<h3>Collect feedback on your model</h3>
```
Comment (alexcombessie) [inline README.md:30]: Between installation and collect feedback, there is a missing section which is "Upload your model from your Python environment" 
You can reuse the same content and illustration as on the website

File: README.md [hunk 5]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  
+  <a align='center' href="https://app.arcade.software/share/zPfpXFt8lRm5XDkaWM85">
+         <img width='600' alt="Interactive demo" src="readme/feedback.png">
+      </a>
+<br/>
+  <br/>
+  <br/>
+  
+  * Collaborate with instant feedback.<br/>
```
Comment (alexcombessie) [inline README.md:39]: This is repetitive with the section title, remove it
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  * Improve ML models with business stakeholders in no time.
```
Comment (alexcombessie) [inline README.md:40]: No need for bullet points here
```suggestion
Improve ML models with business stakeholders in no time.
```

File: README.md [hunk 7]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  
+ <h3>Create Tests from the feedback</h3>
```
Comment (alexcombessie) [inline README.md:42]: Can you reuse the same title as on the website, "Get automated test suites in seconds" ?

File: README.md [hunk 8]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  
+  <a href="https://app.arcade.software/share/zPfpXFt8lRm5XDkaWM85">
+         <img width='600' alt="Interactive demo" src="readme/test.png">
+      </a>
+<br/>
+   <br/>
+   <br/>
+  
+
+  * Get automated test suites in seconds.</br>
```
Comment (alexcombessie) [inline README.md:52]: Repetitive
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  * Exhaustive test suites, backed by 
State-of-the-Art ML research.
```
Comment (alexcombessie) [inline README.md:53]:
```suggestion
Exhaustive test suites, backed by  State-of-the-Art ML research.
```

File: README.md [hunk 10]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  
+  
+ <h3>Workflow</h3>
```
Comment (alexcombessie) [inline README.md:56]: This section is too high level and repetitive, let's remove it

File: README.md [hunk 11]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
+  <br/>
+  
+  * **Upload** - Easy upload for any Python model: PyTorch, TensorFlow, Transformers, sklearn, etc.
+  * **Validate** - Collaborate with business stakeholders with direct feedback & discussion
+  * **Test** - Exhaustive test suites, backed by 
State-of-the-Art ML research.
+  * **Automate** - Protect your ML models against the risk of regressions, drift and bias.
+
+<br/>
+<h3>Requirements</h3>
```
Comment (alexcombessie) [inline README.md:71]: This section should be a subsection of the "Installation" section

File: README.md [hunk 12]
```diff
@@ -28,22 +27,67 @@ http://localhost:19000
...
 
 - [docker](https://docs.docker.com/get-docker/) 
 - [docker-compose](https://docs.docker.com/compose/install/) 
 
-<h3 align="center">Quick Start</h3>
+<h3>Quick Start</h3>
```
Comment (alexcombessie) [inline README.md:76]: This is repetitive, remove this section

**Review by alexcombessie — CHANGES_REQUESTED** [review]

**Comment by mackerac** [comment]
Changes Requested completed

File: README.md (hunk 1 above)
Comment (alexcombessie) [reply README.md]: @mackerac As you can see below, having two lines below titles is odd.
Screenshot 2022-06-22 at 10 40 37

To fix it, can you change ""Deliver ML products, better & faster" to h3?

File: README.md (hunk 7 above)
Comment (alexcombessie) [reply README.md]: Nitpick: can you remove the final period "." in the end of the title? Details matter.

Comment (mackerac) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (mackerac) [reply README.md]: Done

**Review by alexcombessie — CHANGES_REQUESTED** [review]
Small thing before merging

After each section, upload your model, collect feedback, etc., can you put a link to the relevant documentation section?
Ex: Upload your model -> https://docs.giskard.ai/start/guides/upload-your-model
For the style, you could for instance write "👉 Documentation" with a hyperlink.

**Comment by mackerac** [comment]
Added documentation links

**Review by alexcombessie — APPROVED** [review]

**Comment by alexcombessie** [comment]
Good for me! @jmsquare can you review as well?

**Comment by mackerac** [comment]
Changes suggested by JM added in the last commits.

**Pull request merged**